packages/datagateway-download/src/downloadCart/downloadCartTable.component.tsx

```diff
@@ -207,7 +207,16 @@ const DownloadCartTable: React.FC<DownloadCartTableProps> = (
           </Grid>
         )}
         <Grid item>
-          <Paper style={{ height: 'calc(100vh - 150px)' }}>
+          {/* Table should take up page but leave room for: SG appbar, 
+              SG footer, tabs, table padding, text below table, and buttons
+              (respectively). */}
+          <Paper
+            style={{
+              height:
+                'calc(100vh - 64px - 24px - 48px - 48px - 3rem - (1.75 * 0.875rem + 12px)',
```

Comment:
There is still a scroll bar as shown in the screenshot below. I think you will need to subtract a few more pixels from `100vh` like I did in my #535 PR to fix this. You will notice in there that I subtracted `180px` despite the total pixels for the 4 components (AppBar, Footer, cart icon and card toggle) being `174px`. I could not understand why this was happening.

![image](https://user-images.githubusercontent.com/45173816/104172355-0b37c380-53fc-11eb-95d1-e572903b367f.png)

Reply:
Ah I see, that's my fault for not actually testing this myself and relying on unit tests; I assumed adding in the size of the footer was such a simple change it wouldn't break anything...

Had a look into what causes the extra space, and it's the [padding from the routing component](https://github.com/ral-facilities/scigateway/blob/master/src/routing/routing.component.tsx#L25) which is 30px. 

I'll up the padding here to 30, and since you've changed it in dataview I'll have a look at `search` and change the height there as part of this PR as well.

Reply:
Does the same test criteria applies for the **search** plugin changes - _Check no (window) bar appears at reasonable (i.e. >500*500px) browser sizes_? Because the vertical scroll bar appears at 820px window width. Also I noticed (but don't think that it's as a result of the changes that you have made) that the **search** plugin table displays scrollbars at 558px window height but the records can't be seen when you try to scroll down. Reducing the window height below this results in the table height reducing and eventually getting removed.

Reply:
I've made a few more changes, so now the search box/table have a minimum width of 600px (584 + 16 padding), which is the xs/sm screen breakpoint. Below that you will get a horizontal scrollbar. For me nothing jumps onto a new line, but I suspects that's dependent on browser settings. For example when running the plugin standalone there's quite a lot of space to play with, but serving it through SG chooses a different font which is wider and so there's no spare pixels in the layout. Equally increasing the font size (my browser was set to 16) will probably force newlines.

In terms of height, I added a minimum height to the search tables like I did for the downloads, so it should always display 3-4 rows at one time. This now means the vertical scrollbar appears sooner (for me, 662 px when served through SG).

While there's still the potential for this to be broken when upping the browsers font size, I can't think of much more we can do without fundamentally changing how we're laying things out.

![image](https://user-images.githubusercontent.com/61705287/105176719-80527980-5b1d-11eb-956a-e692c724fe39.png)

Reply:
This looks good now, thanks for making the changes.